phing/build.yml

```diff
@@ -58,9 +60,9 @@ drupal:
 
 drush:
   bin: ${composer.bin}/drush
-  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${multisite.name}
+  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${site.uri.${environment}}
```

Comment:
Instead of manually specifying the site URI here, shouldn't that be defined as part of the Drush site alias? There would need to be a separate Drush alias for each site anyway.